src/Repository/MajorVersionRepository.php

```diff
@@ -131,4 +134,17 @@ function (Release $a, Release $b) {
 
         return $releases;
     }
+
+    /**
+     * @param MajorVersion $majorVersion
+     * @return MajorVersion
+     */
+    private function removeEltsReleases(MajorVersion $majorVersion): MajorVersion
+    {
```

Comment:
I meant: use a dedicated method (not a boolean switch) in this repository to find the elts releases, and select the correct descendants directly - so basically a `findSupportedGroupedByMajor` method.